ibm/service/vpc/data_source_ibm_is_backup_policy_job.go

```diff
@@ -148,6 +149,78 @@ func DataSourceIBMIsBackupPolicyJob() *schema.Resource {
 							Computed:    true,
 							Description: "The unique user-defined name for this volume.",
 						},
+						"remote": &schema.Schema{
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "If present, this property indicates that the resource associated with this reference is remote and therefore may not be directly retrievable.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"href": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The URL for this region.",
+									},
+									"name": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The globally unique name for this region.",
+									},
+								},
+							},
+						},
+						"resource_type": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The resource type.",
+						},
+					},
+				},
+			},
+			"source_instance": &schema.Schema{
```

Comment:
This source can be `VolumeReference` or might be `InstanceReference` and additionally if `VolumeRefernce` has `remote` information which I see is missing if we cast it to `VolumeReference`.